src/test/ui/parser/assoc-const-underscore-semantic-fail.stderr

```diff
@@ -20,7 +20,10 @@ error[E0438]: const `_` is not a member of trait `A`
   --> $DIR/assoc-const-underscore-semantic-fail.rs:10:9
    |
 LL |         const _: () = ();
-   |         ^^^^^^^^^^^^^^^^^ not a member of trait `A`
+   |         ^^^^^^^^^^^^^^^^^
+   |         |
+   |         not a member of trait `A`
+   |         help: there is an associated constant with a similar name: `_`
```

Comment:
Can we special case this? Ideally we would completely silence this error (as it is redundant with the earlier ones). Maybe we can do that in a separate PR...